string-manipulation-2/src/first-chars.ts

```diff
@@ -0,0 +1,12 @@
+/* exported firstChars */
+function firstChars(length: number, str: string): string {
+  let newStr = '';
+  if (str.length <= length) {
+    return str;
+  } else {
+    for (let i = 0; i < length && Math.max(0, str.length); i++) {
```

Comment:
condition expr:
i < Math.min(length, str.length)
will ensure no bug if length is more than length of string